tests/test_backend.py

```diff
@@ -1957,6 +1957,75 @@ def test_reverse_sequence_time_major(self):
         _ = tf.identity(x_, name=_TFOUTPUT)
         self._run_test_case([_OUTPUT], {_INPUT: x_val})
 
+
+    @skip_opset(9, "ReverseV2")
```

Comment:
hm.. it doesn't work, please use `@check_opset_min_version(10, "ReverseSequence")`.

Reply:
Replaced `@skip_opset(9, "ReverseV2")` with `@check_opset_min_version(10, "ReverseSequence")` on all 3 test functions.